minimesos/src/main/java/com/containersol/minimesos/cluster/ClusterRepository.java

```diff
@@ -0,0 +1,90 @@
+package com.containersol.minimesos.cluster;
+
+import com.containersol.minimesos.MinimesosException;
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.IOUtils;
+import org.apache.log4j.Logger;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class ClusterRepository {
```

Comment:
I truly don't want to sound like a priest. However, what if we try to javadoc our code